pkg/storaged/stratis-details.jsx

```diff
@@ -171,6 +173,13 @@ export function validate_pool_name(client, pool, name) {
 
 export const StratisPoolDetails = ({ client, pool }) => {
     const filesystems = client.stratis_pool_filesystems[pool.path];
+    const key_desc = (pool.Encrypted &&
+                      pool.KeyDescription[0] &&
+                      pool.KeyDescription[1][1]);
+    const can_tang = (pool.Encrypted &&
+                      pool.ClevisInfo[0] && // pool has consistent clevis config
+                      (!pool.ClevisInfo[1][0] || pool.ClevisInfo[1][1][0] == "tang")); // not bound or bound to "tang"
+    const tang_url = can_tang && pool.ClevisInfo[1][0] ? JSON.parse(pool.ClevisInfo[1][1][1]).url : null;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)